test/commands/devtool.test.js

```diff
@@ -27,46 +27,79 @@ const TIMEOUT = 3000;
 describe("devtool command", function() {
     it("devtool starts and serves html", async function() {
         // set up server
+        const port = 8888;
         stdout.start();
         const devtool = new DevToolCommand([]);
-        devtool.run([]);
+        devtool.run(port);
         await sleep(SERVER_START_UP_WAIT_TIME);
-        const port = devtool.server.address().port;
         stdout.stop();
 
         // check start up logs
         const stdoutList = stdout.output.split('\n');
-        assert.strictEqual(stdoutList[0], `Starting Asset Compute Developer Tool Server on port ${port}`);
-        assert.ok(stdoutList[1], `Listening on port ${port}`);
-        assert(stdoutList[2].includes(`Asset Compute Developer Tool Server started on url http://localhost:${port}/?devToolToken=`));
-        const url = stdoutList[2].split(' ').pop();
+        const regex = new RegExp(`Asset Compute Developer Tool Server started on url  http://localhost:${port}/\\?devToolToken=[a-zA-Z0-9].*`);
+        assert.strictEqual(regex.test(stdoutList[0]), true);
+
+        const url = stdoutList[0].split(' ').pop();
         assert.ok(url.includes(`http://localhost:${port}/?devToolToken=`));
 
+        const token = url.split('=')[1];
+        assert.strictEqual(token.length, 64);
+
         // api call to get raw html
         const resp = await fetch(url);
         assert.strictEqual(resp.status, 200);
         const html = await resp.text();
         assert.ok(html.includes('/static/js'));
-        await devtool.stop();
```

Comment:
we still need to kill the server somehow so it does not take up space on a port after the test

Reply:
I would expect the `stop` call to do that - aka finish server execution properly with resources release.
Why would a user of devtool have to clean up port usage himself? This should be transparent to the caller here.

Reply:
since we now use the api from the devtool to start the server, we no longer have direct control over stopping/starting the server. We cannot use a `stop()` api until there is one added to the devtool. For now so the tests don't hang or leave ports open, we should just kill the node process when we are done with the test